about.html

```diff
@@ -0,0 +1,125 @@
+        <div class="a-headline">
+          <p class="a-ph1">"Stay in the game!"</p>
+          <h1  class="a-p1">
+            <span>Worldwide broadcast E3 2021
+          </h1>
+          <h2>
+            <span class="a-pd1">Worldwide broadcast E3 2021
+          </h2>
+          <p class="a-pheadline">
+            E3 is the world’s premier event for computer and video games and related products. 
+            The show is owned and operated by the ESA, the US trade association dedicated to serving 
+            the business and public affairs needs of the companies developing and publishing 
+            interactive games for video game consoles, handheld devices, personal computers, and the 
+            internet.
+          </p>
+          <p class="a-contact">Press Contacts: Kat Jones, Motiv PR, kjones@motivpr.com
+            and Johner Riehl, Wonacott PR, jriehl@wonacottpr.com</p>
+        </div>
+        <section class="info">
+          <h2 class="infotittle">Electronic Entertainment Expo Logo 2021</h2>
+          <h2 class="eventsline">____</h2>
+          <p class="infodescpt">The E3 logo and variations are trademarks and/or registered trademarks 
+            of the Entertainment Software Association.</p>
+          <div class="infocard"> <img class="logoimg" src="./imgs/E3Logo.png" alt=""></div>
+        </section>
+        <section class="pastinfo">
+          <h2 class="pastinfotittle">See Our Last Events</h2>
+          <h2 class="eventsline">____</h2>
+          <p class="infodescpt">Take a look to the last two years E3 events and all the historical announcements</p>
+          <div class="infobuttoncontainer">
+          <button class="lastinfobutton"><span class="infobuttontext">E3 2019</span><br>
+            <span class="infobuttontext1">2019 was the 25th E3 press conference</span></button>
+          <button class="lastinfobutton1"><span class="infobuttontext">E3 2020</span><br>
+            <span class="infobuttontext1">2020 was the 16th E3 press conference</span></button>
+          </div>
+        </section>
+        <section class="partner">
+          <div>
+          <h2 class="partnertittle">Partner</h2>
+          <h2 class="eventsline">____</h2>
+          </div>
+          <div class="partnerimgs">
+          <img class="partnerimg" src="./imgs/white-nvidia-logo-1png.png" alt="">
+          <img class="partnerimg" src="./imgs/SONY-SQUARE1.png" alt="">
+          <img class="partnerimg" src="./imgs/ubisoft-logo1.png" alt="">
+          <img class="partnerimg" src="./imgs/nintendologo1.png" alt="">
+          <img class="partnerimg" src="./imgs/Electronic_Arts-Logo.png" alt="">
+        </div>
+        </section>
+      </main>
+      <footer class="aboutfooter">
+        <img class="footerimg" src="./imgs/E3logowhite.png" alt="">
+        <h2 class="a-footertext">©Entertainment Software Association 2021<br><br><br>
+          601 Massachusetts Avenue, NW - Suite 300 - Washington, DC 20001</h2>
+      </footer>
```

Comment:
- [ ] I think the footer should be the same for both pages, on desktop size is almost the same except the background-color, please work on this. 🐶
- Hint: Mobile size could have a white background color, and dark for desktop size.
- [ ] There's no footer on home mobile size, please add it. 🐦 
![image](https://user-images.githubusercontent.com/20563252/128289997-1fb21689-5df0-4f3d-8714-9a3ecb9c4a27.png)

Reply:
Perfect! it's done, I just left the background color different in the desktop version for the index and about pages, as you can see in the design guidelines, they have different colors.
INDEX FOOTER
![Design for about](https://user-images.githubusercontent.com/82678907/128385424-1931c0f2-2cfc-47c8-b3ed-9922d92f1154.png)
ABOUT
![Design for Index](https://user-images.githubusercontent.com/82678907/128385400-9289e31d-e380-4d4a-b557-82a6e548234b.png)

The footer was added in mobile. 
![Captura de pantalla 2021-08-05 a la(s) 11 25 45 a m](https://user-images.githubusercontent.com/82678907/128386211-4295a826-67cc-47bf-bcd5-212bed8716db.png)


thanks 😄